.github/workflows/acceptance_tests_reusable.yaml

```diff
@@ -136,6 +136,7 @@ jobs:
 
   deployment_cleanup:
     needs: [ acceptance_tests_verify ]
+    if: "!contains(github.event.pull_request.labels.*.name, 'skip-cleanup')"
```

Comment:
_(this comment is unrelated to the code-line here, just need a place to write a comment)_

Can you add the info to the PR-description that we require the possibility to send envelopes also to syslog-server  because the metron-agent on the bosh-vms aren't reachable when this get's deployed as CF app. It took some minutes for me to understand this and maybe we can improve here when others came across this change.

Reply:
Not sure why you _resolved_ this comment but I don't see any changes in the PR description :/